tests/wpt/css-tests/tools/manifest/sourcefile.py

```diff
@@ -198,6 +198,23 @@ def viewport_size(self):
         return self.viewport_nodes[0].attrib.get("content", None)
 
     @cached_property
+    def dpi_nodes(self):
+        """List of ElementTree Elements corresponding to nodes in a test that
+        specify device pixel ratios"""
+        return self.root.findall(".//{http://www.w3.org/1999/xhtml}meta[@name='device-pixel-ratio']")
```

Comment:
Not sure you absolutely need to specify a specific namespace here, but this is fine.